app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionDetailViewModel.java

```diff
@@ -114,6 +121,21 @@ public void destroy() {
         // Do nothing
     }
 
+    @Nullable
+    private String createSpeakerImageUrl(@Nullable Speaker speaker) {
+        if (speaker == null) {
+            return null;
+        }
+
+        String imageUrl = speaker.imageUrl;
+
+        if (!TextUtils.isEmpty(imageUrl) && imageUrl.startsWith("/")) {
```

Comment:
Some speakers have image_url that isn't start with / (like https://...) , so we shoud handle url type or fix sessions.json.